pkg/parser/terraform.go

```diff
@@ -0,0 +1,46 @@
+package parser
```

Comment:
@insaaniManav Can we please have tests for this please? Once the tests are written, we are good to merge. Refer to the npm parser test case if you need.

https://github.com/safedep/vet/blob/main/pkg/parser/npm_graph_test.go